docker-compose.yml

```diff
@@ -21,8 +21,9 @@ services:
       test: ['CMD', 'curl', '-f', 'http://localhost:8090/auth/health']
     environment:
       - SERVER_PORT=8090
-      - SPRING_PROFILES_ACTIVE=dev
+      - SPRING_PROFILES_ACTIVE=dev,delius
       - SPRING_JPA_PROPERTIES_HIBERNATE_SHOW_SQL=false
+      - DELIUS_ENDPOINT_URL=http://community-api:8080
```

Comment:
Would you mind adding a bit more detail as to why this has been added to the commit message e.g. "to configure the service look up the user from the locally running instance of the community api"?

Reply:
👍 I felt dirty changing somebody else's commit message, but I powered through the guilt and did it